pkg/network/common/validation.go

```diff
@@ -147,12 +147,18 @@ func ValidateHostSubnet(hs *networkapi.HostSubnet) error {
 	for i, egressIP := range hs.EgressIPs {
 		if _, err := validateIPv4(string(egressIP)); err != nil {
 			allErrs = append(allErrs, field.Invalid(field.NewPath("egressIPs").Index(i), egressIP, err.Error()))
+			// if any one of the IPs is invalid, we reset the whole array
+			hs.EgressIPs = nil
```

Comment:
This can also cause downtime. Just remove the invalid ones.

Reply:
> This can also cause downtime. Just remove the invalid ones.

ack yea, I need to update this patch.